po/it.po

```diff
@@ -0,0 +1,196 @@
+   
+#: counter.lua:240 main.lua:318  
+msgid "Name"  
+msgstr "Nome"  
+   
+#: counter.lua:246  
+msgid "Color"  
+msgstr "Colore"  
+   
+#: counter.lua:255  
+msgid "Move to top"  
+msgstr "Sposta in cima"  
+   
+#: counter.lua:258  
+msgid "Move counter to the top of the current list"  
+msgstr "Sposta il contatore in cima alla lista corrente"  
+   
+#: counter.lua:264  
+msgid "Move to bottom"  
+msgstr "Sposta in fondo"  
+   
+#: counter.lua:267  
+msgid "Move counter to the bottom of the current list"  
+msgstr "Sposta il contatore in fondo alla lista corrente"  
+   
+#: counter.lua:282  
+msgid "Move up"  
+msgstr "Sposta su"  
+   
+#: counter.lua:285  
+msgid "Move counter to just above the previous row"  
+msgstr "Sposta il contatore appena sopra la riga precedente"  
+   
+#: counter.lua:292  
+msgid "Move down"  
+msgstr "Sposta giù"  
+   
+#: counter.lua:295  
+msgid "Move counter to just below the next row"  
+msgstr "Sposta il contatore appena sotto la riga successiva"  
+   
+#: counter.lua:327  
+msgid "Create a new window for this counter"  
+msgstr "Crea una nuova finestra per questo contatore"  
+   
+#: counter.lua:334  
+msgid "Show in a separate window"  
+msgstr "Mostra in una finestra separata"  
+   
+#: counter.lua:417 main.lua:140 main.lua:232 main.lua:486  
+msgid "Tally"  
+msgstr "Contatore"  
+   
+#: counter.lua:431  
+msgid "Decrement by 1"  
+msgstr "Decrementa di 1"  
+   
+#: counter.lua:440  
+msgid "Increment by 1"  
+msgstr "Incrementa di 1"  
+   
+#: main.lua:37  
+msgid "No color"  
+msgstr "Nessun colore"  
+   
+#: main.lua:39  
+msgid "Red"  
+msgstr "Rosso"  
+   
+#: main.lua:41  
+msgid "Orange"  
+msgstr "Arancione"  
+   
+#: main.lua:43  
+msgid "Yellow"  
+msgstr "Giallo"  
+   
+#: main.lua:45  
+msgid "Green"  
+msgstr "Verde"  
+   
+#: main.lua:47  
+msgid "Blue"  
+msgstr "Blu"  
+   
+#: main.lua:49  
+msgid "Purple"  
+msgstr "Viola"  
+   
+#: main.lua:146  
+msgid "translator-credits"  
+msgstr "Albano Battistella"  
+   
+#: main.lua:150  
+msgid "Translate this app!"  
+msgstr "Traduci questa app!"  
+   
+#: main.lua:151  
+msgid "Send a tip!"  
+msgstr "Invia una mancia!"  
+   
+#: main.lua:152  
+msgid ""  
+"<p>Various enhancements have been made to allow Tally to be used with a "  
+"screen reader. Keyboard shortcuts for closing counter windows have been "  
+"added.</p>\n"  
+msgstr ""  
+"<p>Sono stati apportati diversi miglioramenti per consentire l'utilizzo di "  
+"Tally con un lettore di schermo. Sono state aggiunte scorciatoie da tastiera "  
+"per chiudere le finestre dei contatori.</p>\n"  
+   
+#: main.lua:157  
+msgid "Main Window"  
+msgstr "Finestra Principale"  
+   
+#: main.lua:160 main.lua:177  
+msgid "Quit Application"  
+msgstr "Esci dall'applicazione"  
+   
+#: main.lua:165 main.lua:200  
+msgid "Keyboard Shortcuts"  
+msgstr "Scorciatoie da Tastiera"  
+   
+#: main.lua:170 main.lua:201  
+msgid "About Tally"  
+msgstr "Informazioni su Tally"  
```

Comment:
Other localizations translate the app name everywhere. For consistency, this should read "Informazioni su Contatore".